src/App/App.js

```diff
@@ -13,6 +11,11 @@ import Quiz from '../Quiz/Quiz'
 import SavedView from '../SavedView/SavedView'
 
 const App = (props) => {
+
+  useEffect(() => {
+    props.setUser({userId: 1})
```

Comment:
Thanks for doing this @saraho1123! I think it makes sense to initialize a user here since we dont have a login yet.